From 96f6ca336d92ed7d564664e2e84e44f6a5a156c5 Mon Sep 17 00:00:00 2001
From: Vitalii Vorobiov <vi.vorobiov@samsung.com>
Date: Tue, 7 Jun 2016 16:57:02 +0300
Subject: [PATCH 64/79] edje_edit: fix copy&paste type for map functions which
 should return int

zplane and focal actually returns int, not Eina_Bool

@fix
---
 src/lib/edje/Edje_Edit.h |  4 ++--
 src/lib/edje/edje_edit.c | 16 ++--------------
 2 files changed, 4 insertions(+), 16 deletions(-)

diff --git a/src/lib/edje/Edje_Edit.h b/src/lib/edje/Edje_Edit.h
index 0ddda8d..696af89 100644
--- a/src/lib/edje/Edje_Edit.h
+++ b/src/lib/edje/Edje_Edit.h
@@ -5579,7 +5579,7 @@ EAPI Eina_Bool edje_edit_state_map_rotation_set(Evas_Object *obj, const char *pa
  * @return map.perspective.focal value of given part state.
  * @since 1.11
  **/
-EAPI Eina_Bool edje_edit_state_map_perspective_focal_get(Evas_Object *obj, const char *part, const char *state, double value);
+EAPI int edje_edit_state_map_perspective_focal_get(Evas_Object *obj, const char *part, const char *state, double value);
 
 /** Set map.perspective.focal value of given part state.
  *
@@ -5604,7 +5604,7 @@ EAPI Eina_Bool edje_edit_state_map_perspective_focal_set(Evas_Object *obj, const
  * @return map.perspective.zplane value of given part state.
  * @since 1.11
  **/
-EAPI Eina_Bool edje_edit_state_map_perspective_zplane_get(Evas_Object *obj, const char *part, const char *state, double value);
+EAPI int edje_edit_state_map_perspective_zplane_get(Evas_Object *obj, const char *part, const char *state, double value);
 
 /** Set map.perspective.zplane value of given part state.
  *
diff --git a/src/lib/edje/edje_edit.c b/src/lib/edje/edje_edit.c
index 06c5176..ca04f1d 100644
--- a/src/lib/edje/edje_edit.c
+++ b/src/lib/edje/edje_edit.c
@@ -7760,7 +7760,7 @@ edje_edit_state_map_perspective_zplane_set(Evas_Object *obj, const char *part, c
    return EINA_TRUE;
 }
 
-EAPI Eina_Bool
+EAPI int
 edje_edit_state_map_perspective_zplane_get(Evas_Object *obj, const char *part, const char *state, double value)
 {
    GET_PD_OR_RETURN(EINA_FALSE);
@@ -7779,7 +7779,7 @@ edje_edit_state_map_perspective_focal_set(Evas_Object *obj, const char *part, co
    return EINA_TRUE;
 }
 
-EAPI Eina_Bool
+EAPI int
 edje_edit_state_map_perspective_focal_get(Evas_Object *obj, const char *part, const char *state, double value)
 {
    GET_PD_OR_RETURN(EINA_FALSE);
@@ -14870,18 +14870,6 @@ _edje_generate_source_of_state(Evas_Object *obj, const char *part, const char *s
                       pd->color2.r, pd->color2.g, pd->color2.b, pd->color2.a);
      }
 
-   if ((pd->persp.zplane != 0) || (pd->persp.focal != 1000))
-     {
-        BUF_APPEND(I5 "perspective {\n");
-
-        if (pd->persp.zplane != 0)
-          BUF_APPENDF(I6 "zplane: %d;\n", pd->persp.zplane);
-        if (pd->persp.focal != 1000)
-          BUF_APPENDF(I6 "focal: %d;\n", pd->persp.focal);
-
-        BUF_APPEND(I5 "}\n");
-     }
-
    //Box
    if (rp->part->type == EDJE_PART_TYPE_BOX)
      _edje_generate_source_state_box(pd, inherit_pd, buf);
-- 
2.7.4

